finality-aleph/src/justification/compatibility.rs

```diff
@@ -57,12 +57,17 @@ enum VersionedAlephJustification {
 }
 
 fn encode_with_version(version: Version, mut payload: Vec<u8>) -> Vec<u8> {
-    let mut result = version.encode();
+    let mut result =
```

Comment:
nit: extract this function to deduplicate code

Reply:
I would have to create one common file or folder for compatibility, I'm not sure it is worth it now with only two places that use this code. I would leave it as it is right now.